packages/connectivity_plus/connectivity_plus/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-all.zip
```

Comment:
I guess you don't know what is the difference between `bin` and `all`, since it is definitely doesn't change anything about the problem you described in the issue.